packages/atomic/cypress/e2e/result-list/folded-result-list.cypress.ts

```diff
@@ -1,219 +1,13 @@
-      .should('have.attr', 'slot', afterChildrenSlotName);
-
-    FoldedResultListSelectors.resultChildren()
-      .shadow()
-      .find('slot')
-      .eq(1)
-      .should('have.attr', 'name', afterChildrenSlotName);
-
-    FoldedResultListSelectors.childResultAtIndex(1)
-      .find(resultChildrenComponent)
-      .should('not.exist');
-  });
-
-  describe('when grandchildren templates are specified', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .with(setSourceAndSortCriteria)
-        .with(
-          addFoldedResultList(
-            buildTemplateWithoutSections(
-              buildResultTopChild(buildResultChildren())
-            )
-          )
-        )
-        .init();
-    });
-
-    assertRendersGrandchildren();
-  });
-
-  describe('when result children component is set to inherit templates and no template is passed', () => {
-    beforeEach(() => {
-      const resultChildren = generateComponentHTML(resultChildrenComponent);
-      resultChildren.setAttribute('inherit-templates', 'true');
-      new TestFixture()
-        .with(setSourceAndSortCriteria)
-        .with(
-          addFoldedResultList(
-            buildTemplateWithSections({
-              title: generateComponentHTML(resultLinkComponent),
-              children: buildResultChildren(resultChildren),
-            })
-          )
-        )
-        .init();
-    });
-
-    assertRendersGrandchildren();
-  });
-
-  describe('should allow to load a folding collection', () => {
-    function setupCollection() {
-      const resultChildren = generateComponentHTML(resultChildrenComponent);
-      resultChildren.setAttribute('inherit-templates', 'true');
-      new TestFixture()
-        .with(setSourceAndSortCriteria)
-        .with(
-          addFoldedResultList(
-            buildTemplateWithoutSections(buildLoadMoreChildren())
-          )
-        )
-        .init();
-
-      FoldedResultListSelectors.loadMoreChildren().click();
-      cy.wait(TestFixture.interceptAliases.UA);
-    }
-
-    beforeEach(() => {
-      setupCollection();
-    });
-
-    assertRendersWholeCollection();
-
-    describe('should restore initial results when collapsing collection', () => {
-      beforeEach(() => {
-        FoldedResultListSelectors.collapseButton().click();
-        cy.wait(200);
-      });
-
-      assertRendersGrandchildren();
-    });
-  });
-
-  describe('with an invalid configuration', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .with(setSourceAndSortCriteria)
-        .with(
-          addFoldedResultList(undefined, {
-            'child-field': '',
-            'collection-field': '',
-          })
-        )
-        .init();
-    });
-
-    CommonAssertions.assertConsoleError();
-    CommonAssertions.assertContainsComponentError(
-      FoldedResultListSelectors,
-      true
-    );
-  });
-});
+tests(
+  FoldedResultListSelectors,
+  foldedResultListComponent,
+  addResultList,
+  'Folded Result List Component'
+);
```

Comment:
@louis-bompart Hi, every result list component is hard failing in cypress 13 so I am looking further into them. I think this is a mistake, in the foldedResultList test suite you are using the **atomic-result-list** component. There is a completely different component that is supposed to be tested in this suite called **atomic-folder-result-list**. I might be wrong tho, can you explain why you used **addResultList** ?